helm/qontract-reconcile/values-internal.yaml

```diff
@@ -204,6 +204,7 @@ integrations:
       memory: 200Mi
       cpu: 200m
 - name: gitlab-housekeeping
+  sleep_duration_secs: "0"
```

Comment:
This look like this will run continually (no sleep). Is this desired?

Reply:
GitLab housekeeping can only merge one MR for app-interface in one loop. The faster it iterates, the more quickly MR can be merged. In the case of foolproof, I have changed the default value in template to 600 and override the parameter in the saas file.